Stetskyi_Homework_7/Unit Testing/ImplementedKatas/FizzBuzz/FizzBuzz/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace FizzBuzz
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            for (int i = 1; i < 100; i++)
+            {
+                Console.WriteLine(WriteFizzBuzz(i));
+            }
+        }
+
+        public static string WriteFizzBuzz(int myint)
+        {
+            ExceptionHandler(myint);
+
+
+            if (myint % 3 == 0 && myint % 5 == 0)
+            {
+                return "FizzBuzz";
+            }
+            else if (myint % 3 == 0)
+            {
+                return "Fizz";
+            }
+            else if (myint % 5 == 0)
+            {
+                return "Buzz";
```

Comment:
@IhorStetskyi, extract to local const, please.

Reply:
numbers 3 & 5 extracted to local consts multiplyToThree & multiplyToFive